Doc/library/ctypes.rst

```diff
@@ -2391,6 +2391,16 @@ Structured data types
    Abstract base class for unions in native byte order.
 
 
+.. class:: BigEndianUnion(*args, **kw)
```

Comment:
these two need versionadded 3.11 markers in the docs.

Reply:
Agreed.